.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: php
 
+sudo: false
+
 matrix:
   allow_failures:
     - php: hhvm
 
 php:
-  - 5.3
```

Comment:
As of version 1.0, this library does not support php 5.3. Proof: composer.json is set to require php >=5.4 and code like https://github.com/schmittjoh/serializer/blob/1.0.0/src/JMS/Serializer/Serializer.php#L82 is not supported in 5.3.

I know it is not related to my PR, but it broke a lot of tests